src/makeCalculator.js

```diff
@@ -4,7 +4,32 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculator = {
+    add: (num) => result + num,
+    subtract: (num) => result - num,
+    multiply: (num) => result * num,
+    divide: (num) => (num === 0 ? result : result / num),
```

Comment:
[CODE STYLE] - Use arithmetic operators with assignment operator `=`. Instead of returning `result / num`, you should update `result` with `result /= num` (taking into account the division by zero case).